WebApp/src/uk/ac/exeter/QuinCe/data/Export/ExportOption.java

```diff
@@ -242,6 +251,12 @@ private void parseJson(ResourceManager resourceManager, JSONObject json)
       }
     }
 
+    if (json.has("visible")) {
+      visible = json.getBoolean("visible");
+    } else {
+      visible = true;
```

Comment:
Since you've set the default value in the variable declaration at the top, there's no need to explicitly set it here.